### PR TITLE
Apiv2 re work

### PR DESCRIPTION
**Pull request by DenizCankurtaran**
WebApps now have different icons depending on the category.
image

### CODE REVIEW

File: src/components/Settings/Profile/Profile.vue [hunk 1]
```diff
@@ -41,8 +41,8 @@
             dense
             solo
             v-model="selected_namespace"
-            :items="[current_user['k8s_namespace']]"
-            :label="current_user['k8s_namespace']"
+            :items="current_user['k8s_accounts'].map(acc => acc['namespace'])"
```
Comment (DenizCankurtaran) [inline src/components/Settings/Profile/Profile.vue:44]: @troeger currently I display only the namespace, but I could change it to service_account:namespce, just like the admin backend, which would you prefer?

Comment (troeger) [reply src/components/Settings/Profile/Profile.vue:44]: Namespace is good enough, people anyway never get the idea of service accounts.

**Comment by troeger** [comment]
How should I review this?

**Comment by DenizCankurtaran** [comment]
> How should I review this?

This is the current design for news
Screenshot from 2021-02-24 12-50-04

**Comment by DenizCankurtaran** [comment]
I will merge the branch for now and work on implementing the new HATEOS api. If any problems arise, or you don't like the new design, please open new Issues. @troeger @kat-hi

**Pull request merged**